plotst/axis.typ

```diff
@@ -143,10 +145,21 @@
       pos.at(1) = -length + pos.at(1)
       invert_markings = -1
     }
-    
+    let arrow_size = 0pt
+    if axis.show_arrows {
+      // sets the size of arrows
+      arrow_size = 2pt
+    }
     if is_vertical(axis) {
       // Places the axis line
-      place(dx: pos.at(0), dy: pos.at(1), line(angle: -90deg, length: length, stroke: axis.stroke))
+      place(dx: pos.at(0), dy: pos.at(1), line(angle: -90deg, length: length - arrow_size * 2, stroke: axis.stroke))
+      
+      // draw the arrow at the end of the axis
+      if axis.show_arrows {
+        place(dx: pos.at(0), dy: pos.at(1) - length, 
+          polygon(fill: axis.stroke, (-arrow_size, arrow_size * 2), (0pt, -1pt), (arrow_size, arrow_size * 2))
```

Comment:
I feel like arrow_size should control the size of the whole arrow and not just half of it.

Reply:
As I think here this links to the previous comment, I would prefer to not make the arrow size adoptable as it gives too much fine control over the appearance, that the end user does not need. I prefer to keep the size adjustable in the `axis.typ` if someone really wants to edit it. Maybe using global constants? But this should be defined globally for the whole project, as different styles of graphs don not really make sense in one document. What are you thoughts about that?

Reply:
I've seen one globals implementation with `state` and didn't quite like them to read and write, but maybe can look again into it. It is lighter than having to write out a whole axis / deconstruct your own global object each time.